client/router_create.go

```diff
@@ -62,7 +62,7 @@ func (cli *VanClient) GetVanControllerSpec(options types.SiteConfigSpec, van *ty
 	van.Controller.Replicas = 1
 	//TODO: change these to types constants
 	van.Controller.Labels = map[string]string{
-		"application":          "skupper",
+		"application":          "skupper-service-controller",
```

Comment:
The original idea with the 'application' label was to have one way to select all objects related to skupper. The skupper.io/component would then distinguish specific parts. We may need to change this but it would be good to retain some label common to all.